core/src/storage/device.py

```diff
@@ -95,16 +96,27 @@ def get_device_id() -> str:
     return dev_id.decode()
 
 
-def get_rotation() -> int:
+def get_rotation() -> DisplayRotation:
     rotation = common.get(_NAMESPACE, _ROTATION, public=True)
     if not rotation:
-        return 0
-    return int.from_bytes(rotation, "big")
+        return DisplayRotation.North  # Default to North if no rotation is set
+
+    value = int.from_bytes(rotation, "big")
+    if value == 0:
+        rotation = DisplayRotation.North
+    elif value == 90:
+        rotation = DisplayRotation.East
+    elif value == 180:
+        rotation = DisplayRotation.South
+    elif value == 270:
+        rotation = DisplayRotation.West
+    else:
+        raise ValueError(f"Unsupported display rotation: {rotation}")
```

Comment:
in this case, when loading an invalid value from storage, we don't want to crash but instead use the default

Reply:
https://github.com/trezor/trezor-firmware/pull/4125/commits/5d07343078852d6572a53026f2a3f683e3dddc43